frontend/packages/ceph-storage-plugin/integration-tests/tests/storage-dashboard.scenario.ts

```diff
@@ -0,0 +1,34 @@
+import { browser } from 'protractor';
+import { appHost } from '@console/internal-integration-tests/protractor.conf';
+import { dashboardIsLoaded } from '../../../console-shared/src/test-views/dashboard-shared.view';
+import { serviceName, clusterHealth, clusterName, allNodes } from '../views/storage-dashboard.view';
+import { execSync } from 'child_process';
+
+describe('Check data on Storage Dashboard.', () => {
+  beforeAll(async () => {
+    await browser.get(`${appHost}/dashboards/persistent-storage`);
+    await dashboardIsLoaded();
+  });
+
+  it('Check cluster health is OK', async () => {
+    expect(clusterHealth.getText()).toContain('is healthy');
+  });
+
+  it('Check service name is OCS', async () => {
+    expect(serviceName.getText()).toEqual('OpenShift Container Storage');
```

Comment:
Do we want to match the case of `openshift container storage` that should be good?
Also, put `OpenShift Container Storage` as a constant on top of this file.
```suggestion
    expect(serviceName.getText()).toEqual('OpenShift Container Storage');
```

Reply:
Yes, I think OpenShift Container Storage should be matched case-sensitively because it's our brand, we need to spell it consistently including the capitalization.
Made it a constant on top of the file.